cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,85 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import faker from 'faker';
 
-describe('Settings page', () => {
-  before(() => {
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
 
-  });
+describe('Settings page', () => {
+  let user;
 
   beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
```

Comment:
Be cautious about the usage of stubs and mocks in tests. The database is being cleared before every test which can make the tests more brittle.

Reply:
done